src/icon/internal.tsx

```diff
@@ -17,18 +17,18 @@ type InternalIconProps = IconProps &
   };
 
 function iconSizeMap(height: number) {
-  if (height) {
-    if (height >= 50) {
-      return 'large';
-    } else if (height >= 36) {
-      return 'big';
-    } else if (height >= 24) {
-      return 'medium';
-    } else if (height <= 16) {
-      return 'small';
-    } else {
-      return 'normal';
-    }
+  if (height === 0) {
+    return 'normal';
+  } else if (height >= 50) {
+    return 'large';
+  } else if (height >= 36) {
+    return 'big';
+  } else if (height >= 24) {
+    return 'medium';
+  } else if (height <= 16) {
```

Comment:
minor: Shall we just extend this condition to have `height <= 16 && height > 0`. It would mean -1 branch + smaller change.